phone_calls/nodejs/queries.js

```diff
@@ -286,11 +283,9 @@ async function executeQuery5(question, transaction) {
         Math.round(secondResult) +
         " seconds.\n";
 
-    result.push(secondResult);
-
     printToLog("Result:", output);
 
-    return result;
+    return [firstResult, secondResult];
```

Comment:
Weird thing, for some reason this function was executing twice in parallel, and the result occasionally was `[firstResult, firstResult, secondResult, secondResult]`. I don't know enough about nodejs and asyncs, and nothing jumped at me as obviously erroneous, so the next best thing was making this query return its results without accumulation, as the other queries do.

Reply:
Ah, that'll be because of some ~~lovely~~ JS magic.

If we look at line 253 (prior to this PR):
```js
result = [];
```
Because we didn't declare a local variable using `let` (or `const`), we've just declared a _global_ variable that lives in the global scope of the Node.js application.

That means that if we run `executeQuery5` twice in parallel, a couple of different things can happen:

1. Execution 1 runs `firstQuery` -> Execution 1 runs `secondQuery` -> Execution 2 runs `firstQuery` -> Execution 2 runs `secondQuery`
2. Execution 1 runs `firstQuery` -> Execution 2 runs `firstQuery` -> Execution 1 runs `secondQuery` -> Execution 2 runs `secondQuery`
3. Execution 1 runs `firstQuery` -> Execution 2 runs `firstQuery` -> Execution 2 runs `secondQuery` -> Execution 1 runs `secondQuery`

There are more possibilities even than the above, but that's already enough to show we get non-determinism - and how we get `[firstResult, firstResult, secondResult, secondResult]`. Execution 1 will return that answer in scenario (3) above, because both execution 1 and execution 2 have written their answers to the `results` array, resulting in an array of length 4.

The correct fix here is to change line 253 to
```
let result = [];
```
While we're at it, for hygiene's sake, let's also change `firstQuery` and `secondQuery` declarations - they should both be `const`, ie: `const firstQuery = [ ... ]`.

(This is one of the reasons we like TypeScript, by the way - its compiler catches things that are _allowed_ in JavaScript, but often lead to weird errors, like declaring variables in the global scope!)